src/github/pullRequestManager.ts

```diff
@@ -372,13 +373,20 @@ export class PullRequestManager implements vscode.Disposable {
 
 	}
 
-	get activePullRequest(): (PullRequestModel | undefined) {
-		return this._activePullRequest;
+	get activePullRequest(): PullRequestModel | undefined {
+		if (issueAsPullRequest(this._activeGithubItem)) {
+			return this._activeGithubItem;
+		}
+		return undefined;
+	}
+
+	get activeItem(): (IssueModel | undefined) {
+		return this._activeGithubItem;
 	}
 
-	set activePullRequest(pullRequest: (PullRequestModel | undefined)) {
-		this._activePullRequest = pullRequest;
-		this._onDidChangeActivePullRequest.fire();
+	set activeItem(issue: (IssueModel | undefined)) {

```

Comment:
For pull requests, the `activePullRequest` represents the one that is associated with the branch you're currently on/one you have checked out. Is there an analog for issues? When would this be set?

Reply:
Oh, I guess the idea is that you could have a branch/PR specifically targeting an issue, so there would be a similar notion of it being checked out? In that case, it seems like you could have both an active pull request and issue at the same time, so perhaps exposing `activeIssue` instead would be better?

Reply:
I was thinking of it more as the branch you have checked out would only have one github item associated with it. You're right though that it could have multiple. For now, since it doesn't change existing behavior, can we stick with only having one item associated with it?

Reply:
If this is meant to only track the active issue, let's name it `activeIssue`. If it's meant to be for both PRs and issues, then I think it makes `activePullRequest` unclear and redundant. This would be especially confusing if we did support having an active issue and an active pull request at the same time. IMO it should be the responsibility of the calling code to use either the active issue or active pull request as needed

Reply:
Changed to have `activeIssue` and `activePullRequest`